sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -1105,6 +1105,97 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
     }
   }
 
+  test("INSERT INTO with user specified columns and defaults: positive tests") {
+    withTable("t") {
+      sql("create table t(i boolean default true, s bigint default 42) using parquet")
+      sql("insert into t (i, s) values (true, default)")
+      checkAnswer(sql("select s from t where i = true"), Seq(42L).map(i => Row(i)))
```

Comment:
```suggestion
      checkAnswer(spark.table("t"), Row(true, 42))
```

Reply:
Let's verify the result in this way

Reply:
Thanks, this is better! I updated the existing positive tests for INSERT INTO commands (without user-specified columns) to verify the results in this way as well.